ruma-events-macros/src/event_enum.rs

```diff
@@ -8,41 +8,38 @@ use syn::{
     Attribute, Expr, ExprLit, Ident, Lit, LitStr, Token,
 };
 
-fn room_event_kind(kind: &EventKind, var: &EventKindVariation) -> bool {
-    matches!(kind, EventKind::Message(_) | EventKind::State(_))
-        && !matches!(var, EventKindVariation::Stripped | EventKindVariation::RedactedStripped)
-}
-
-fn room_id_kind(kind: &EventKind, var: &EventKindVariation) -> bool {
-    matches!(kind, EventKind::Message(_) | EventKind::State(_))
-        && matches!(var, EventKindVariation::Full | EventKindVariation::Redacted)
-}
-
-fn sender_id_kind(kind: &EventKind, _var: &EventKindVariation) -> bool {
-    matches!(kind, EventKind::Message(_) | EventKind::State(_) | EventKind::ToDevice(_))
-}
-
 fn prev_content_kind(kind: &EventKind, var: &EventKindVariation) -> bool {
     matches!(kind, EventKind::State(_))
         && matches!(var, EventKindVariation::Full | EventKindVariation::Stub)
 }
```

Comment:
Is this still used somewhere?

Reply:
In the generation of accessor methods since `content` and `prev_content` have to be built differently than the rest.

Reply:
Ah right, I think I get it. Can you just rename to `has_prev_content`?